src/main/java/com/google/api/codegen/transformer/csharp/CSharpApiMethodTransformer.java

```diff
@@ -166,36 +172,11 @@ protected void setServiceResponseTypeName(
           for (FlatteningConfig flatteningGroup : methodConfig.getFlatteningConfigs()) {
             GapicMethodContext methodContext =
                 context.asFlattenedMethodContext(requestMethodContext, flatteningGroup);
-            apiMethods.add(
```

Comment:
This code block has been moved, unchanged, to the body of `generateFlattenedLroMethods()`